palace/utils/geodata.cpp

```diff
@@ -61,7 +62,7 @@ std::unique_ptr<mfem::ParMesh> ReadMesh(MPI_Comm comm, const IoData &iodata, boo
                                         Timer &timer)
 {
   // On root, read the serial mesh (converting format if necessary), and do all necessary
-  // serial preprocessing. When finished, distribute the mesh to all processes. Count disk
+  // serial preprocessing. When eval, distribute the mesh to all processes. Count disk
```

Comment:
Unsure of this change

Reply:
Find and replace error, fixed in changes to appear.